css/style.css

```diff
@@ -49,65 +49,6 @@ body {
   margin: 0 auto;
 }
 
-/*---------- FORMS ----------*/
-
-input[type="text"],
```

Comment:
why are these deleted @RahulBansal0502

Reply:
why are these changes made in the file 
Sorry got wrapped up with some work

Reply:
Actually in our website we are not using any type of form where user will have to enter anything so there is no need of forms.

Reply:
okay, makes sense i thought they were unintentional will review this as well in some time, I confirmed that css file is indeed a dead code, great work noticing that

Reply:
Ya sure thanks!